mava/configs/system/q_learning/rec_iql.yaml

```
@@ -0,0 +1,33 @@
+# --- Defaults REC-IDQN ---
+total_timesteps: 5_000_000
+num_updates: 1000 # total_timesteps takes preference
+seed: 1
+
+# --- Agent observations ---
+add_agent_id: True
+
+# --- RL hyperparameters ---
+buffer_min_size: 32
```

very nit: min buffer size reads a bit better
```suggestion
min_buffer_size: 32
```

---

Done! Didn't commit directly because it needed to be changed elsewhere as well